package.json

```
@@ -36,7 +36,7 @@
   "peerDependencies": {
     "@apollo/client": "^3.2.3",
     "@sentry/browser": "^6.0.4",
-    "graphql": "^15.4.0",
+    "graphql": ">=15.4.0",
```

```suggestion
    "graphql": "15 - 16",
```

We cannot know if we will be compatible with future versions beyond 16.

---

Maybe in that case someone can change it or submit a PR that fixes issues with GQL 17 etc. I'm testing with 16 right now and it seems fine so far.

`apollo-link-sentry` was last released a year ago, so I'm assuming we want to err on the side of requiring less active maintenance.

---

Isn't it possible to ignore mismatched peer dependencies through npm?